src/main/java/info/fingo/urlopia/api/v2/automatic/vacation/days/AutomaticVacationDayService.java

```diff
@@ -49,23 +49,23 @@ public void addHoursForNewYear(){
     }
 
     public AutomaticVacationDay resetPropositionFor(Long userId){
```

Comment:
Minor: Nie wiem czy jest tu jakiś linter, ale w pierwotnych deklaracjach funkcji są spacje przed otwierąjacymi nawiasami.

Reply:
Lintera nie ma, a z tego co widzę, to w całym projekcie jest w tym zakresie totalnie wolna amerykanka, raz jest tak, raz tak.